src/Databases/IDatabase.h

```diff
@@ -170,7 +170,15 @@ class IDatabase : public std::enable_shared_from_this<IDatabase>
     /// Get the table for work. Return nullptr if there is no table.
     virtual StoragePtr tryGetTable(const String & name, ContextPtr context) const = 0;
 
+    using DatabasePtr = std::shared_ptr<IDatabase>;
+    using Databases = std::map<String, DatabasePtr>;
```

Comment:
we already have these aliases declared below - I think we can just bring them up